src/view/com/lightbox/ImageViewing/components/ImageItem/ImageItem.android.tsx

```diff
@@ -42,7 +38,112 @@ type Props = {
   doubleTapToZoomEnabled?: boolean
 }
 
-const AnimatedImage = Animated.createAnimatedComponent(Image)
+ type Matrix3 = readonly [
+  number,
+  number,
+  number,
+  number,
+  number,
+  number,
+  number,
+  number,
+  number
+];
+
+function createTransform(): Matrix3 {
+  'worklet'
+  return [1, 0, 0, 0, 1, 0, 0, 0, 1];
+}
+
+function prependTranslate(t: Matrix3, x: number, y: number) {
```

Comment:
Wrote these as in-place mutations to avoid allocating a bunch of stuff during gestures